DEVELOPMENT.md

```diff
@@ -59,22 +61,22 @@ We'll start by enabling Emacs package repositories and declaring which packages
  (push 'lsp-treemacs package-selected-packages))
 ```
 
-To avoid restarting Emacs, you can evaluate that previous s-expression by pointing at the last parenthesis and using `C-x C-e` or selecting the block and using `M-x eval-region`.
+To avoid restarting Emacs, you can evaluate that previous s-expression by pointing at the last parentheses and using `C-x C-e` or selecting the block and using `M-x eval-region`.
```

Comment:
This is actually different, it's really pointing at the last (singular) `parenthesis`.

Unlike the item below which refers to inserting (plural) `parentheses` and `brackets`.